.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        // os: [ubuntu-latest, macos-latest, windows-latest]
+        # os: [ubuntu-latest, macos-latest, windows-latest]
         os: [ubuntu-latest]
         node: [10, 12]
```

Comment:
I would say even minimum node would be enough 😅 Nuxt node compatibility is out of this module scope

Reply:
Sorry rs